tests/lib/eslint.js

```diff
@@ -2193,6 +2193,11 @@ describe("eslint", function() {
             assert.equal(messages.length, 0);
         });
 
+        it("should send sourceType \"module\" to escope when there is an ecmaFeatures.modules (plural) and ecmaFeatures.blockBindings attributes in the config", function() {
```

Comment:
This test name is incorrect. escope isn't being tested here - you are just testing that the code parses correctly. escope is an implementation detail that is completely hidden from this test.

The test name should be something like "should properly parse import statements when ecmaFeatures.modules is true"